src/components/Pokemon/FavoritePokemon.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import type { Pokemon } from '../../composable/getPokemon'
+import PokemonList from './PokemonList.vue'
+const props = defineProps<{
+  pokemon: Pokemon[] | undefined
+  favorites: Set<number>
+}>()
+const emit = defineEmits(['onLike'])
+
+const favoritePokemon = computed(() => {
+  if (props.pokemon) {
+    return props.pokemon.filter((creature) => {
+      return props.favorites.has(creature.id)
+    })
+  } else {
+    return []
+  }
+})
```

Comment:
This is fine, but could be shortened to 
`const favoritePokemon = computed(() => 
  props.pokemon ? props.pokemon.filter(creature => props.favorites.has(creature.id)) : []
);
`